src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -480,6 +523,55 @@ export default class OrgPerson extends Mixins(CommonMixin) {
     }
   }
 
+  /** Apply input field validations. */
+  applyValidation (): void {
```

Comment:
I kinda like having all validations in the same place here.

I also like using `:rules="isValidation ? myRule : []"`.

Reply:
Indeed, i do as well. 
In this scenario, it seemed easy to apply in one place but both are "valid" :)